### PR TITLE
2.3.4 release notes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -6,6 +6,28 @@ keywords: portworx, release notes
...
+| PWX-10726 | Portworx can now be installed on OpenShift 4.3 when coupled with [Portworx Operator 1.1.1](https://github.com/libopenstorage/operator/releases). |
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-10974 | In version 2.3.3, Portworx erroneously showed multi-part licenses as "expired" when the license with the earliest expiration date expired. Despite this incorrect reporting, multi-part licenses did not expire and the cluster continued functioning normally. <br/><br/>**User Impact:** Users may have seen multi-part licenses erroneously marked "expired". <br/><br/>**Resolution:** Portworx now correctly displays multi-part license expiration dates. |
+| PWX-10967 | PX-Migrate erroneously indicated success when migrating between clusters with the same internal IP addresses. <br/><br/>**User Impact:** Migrations under these circumstances failed, but users saw Portworx indicate success. <br/><br/>**Resolution:** PX-Migrate no longer migrates between nodes with the same internal IP address during migration operations. |
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md]: Resolution is wrong: Px-migrate will now detect internal IPs and ignore internal IPs when migrating

Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @disrani-px

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -6,6 +6,28 @@ keywords: portworx, release notes
...
+|----|----|
+| PWX-10726 | Portworx can now be installed on OpenShift 4.3 when coupled with [Portworx Operator 1.1.1](https://github.com/libopenstorage/operator/releases). |
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-10974 | In version 2.3.3, Portworx erroneously showed multi-part licenses as "expired" when the license with the earliest expiration date expired. Despite this incorrect reporting, multi-part licenses did not expire and the cluster continued functioning normally. <br/><br/>**User Impact:** Users may have seen multi-part licenses erroneously marked "expired". <br/><br/>**Resolution:** Portworx now correctly displays multi-part license expiration dates. |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @zoxpx you might want to take a look.

File: content/reference/release-notes/portworx.md [hunk 3]
```diff
@@ -6,6 +6,28 @@ keywords: portworx, release notes
...
+
+February 3, 2020
+
+### Improvements
+
+Portworx has upgraded or enhanced functionality in the following areas:
+
+|**Improvement Number**|**Improvement Description**|
+|----|----|
+| PWX-10726 | Portworx can now be installed on OpenShift 4.3 when coupled with [Portworx Operator 1.1.1](https://github.com/libopenstorage/operator/releases). |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @piyush-nimbalkar this improvement is yours, let me know if it looks good.

Comment (piyush-nimbalkar) [reply content/reference/release-notes/portworx.md]: Looks good

File: content/reference/release-notes/portworx.md [hunk 4]
```diff
@@ -6,6 +6,28 @@ keywords: portworx, release notes
...
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-10974 | In version 2.3.3, Portworx erroneously showed multi-part licenses as "expired" when the license with the earliest expiration date expired. Despite this incorrect reporting, multi-part licenses did not expire and the cluster continued functioning normally. <br/><br/>**User Impact:** Users may have seen multi-part licenses erroneously marked "expired". <br/><br/>**Resolution:** Portworx now correctly displays multi-part license expiration dates. |
+| PWX-10967 | PX-Migrate erroneously indicated success when migrating between clusters with the same internal IP addresses. <br/><br/>**User Impact:** Migrations under these circumstances failed, but users saw Portworx indicate success. <br/><br/>**Resolution:** Px-migrate now successfully migrates between clusters with the same internal IP addresses. |
```
Comment (disrani-px) [inline content/reference/release-notes/portworx.md]:
```suggestion
| PWX-10967 | PX-Migrate erroneously indicated success when migrating volumes between clusters with the same internal IP addresses. <br/><br/>**User Impact:** Migrations under these circumstances failed, but users saw Portworx indicate success. <br/><br/>**Resolution:** Px-Migrate now successfully migrates volumes between clusters with the same internal IP addresses. |
```

**Pull request merged**